status/types/network_type.nim

```diff
@@ -27,11 +25,3 @@ proc toChainId*(self: NetworkType): int =
     of NetworkType.XDai: result = XDai
     of NetworkType.Poa: result = 99
     of NetworkType.Other: result = -1
-
-proc toNetwork*(self: NetworkType): Network =
-  for network in status_network.getNetworks():
-    if self.toChainId() == network.chainId:
-      return network
-
-  # Will be removed, this is used in case of legacy chain Id
-  return Network(chainId: self.toChainId())
```

Comment:
I had to move this function to network.nim, otherwise I had a recursive dependency, since network now needs settings, settings needs network_type, and network_type somehow needed network.

Reply:
https://c.tenor.com/4EYaYAMus0cAAAAd/michael-scott-the-office.gif